### DISCUSSION TITLE
Welcome to Coding-ninjas-competitive Discussions!

### DISCUSSION THREAD

**Discussion by Mehulcoder** [Mehulcoder/Coding-ninjas-competitive#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by knockcat** [comment]
Mehul I made two new pull request today, please have a look over them.
1.Get_Path_DFS.cpp
2.Connected_Component_DFS.cpp

**Comment by CodeKerdos123** [comment]
CodeKerdos is India’s go-to e-learning platform for practical, career-driven coding education — with live sessions, interactive courses, and real outcomes in both Hindi and English.
We don’t just teach code — we build careers through:
      Hands-on training with live projects and AI-driven tech stacks
      1:1 mentorship from top industry experts
      Career coaching with mock interviews, resume reviews & soft skills
      AI-infused curriculum designed for real-world industry readiness
      Line-by-line clarity: understand each command before you run it

Contact us on hello@codekerdos.in for more information.
